_posts/2018-09-29-cfp-in-progress.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Call for Proposals in progress, keynotes announcement"
+authors:
+  - skade
+  - apiraino
+
+---
+
+# Call for Proposals
+
+Last week we closed the CfP (Call for Proposals) submission window, we allowed a couple more days to really give everyone a chance to submit.
+
+We received 50 interesting proposals to fill up the talks slots for the upcoming Rustfest in Rome, thank you so much! Now it's our turn to open the envelopes and choose those that will make to the conference. As usual, the choice will be tough, all the topics submitted looks absolutely .
+
+Like every year, there's a preliminary selection; here is a draft of the schedule, to be refined and finalized:
+
+| Talk type     | amount planned |
+| ------------- | --------------:|
+| Keynotes      | 2              |
+| 15min format  | 2              |
+| 30min format  | 7              |
+| 45min format  | 2              |
+
+# Keynotes
+
+We are excited to announce the line-up of the speakers for the keynotes:
+
+### Aaron Turon
+
+He works at Mozilla on the Rust programming language, together with an incredible, worldwide community of contributors.
+
+He used to be an academic, studying concurrency, verification and programming language design.
+
+Currently living in Portland, Oregon.
+
```

Comment:
What we are missing here is that we want to turn the second keynote into a project presentation and the three speakers below will present. This is not covered at all.

Reply:
> This time around, we want to try something new: we'll do away with the second keynote. Instead, it will be replaced by what we call the "Project Highlight". For the first iteration, we're going to put [Increasing Rust Reach](https://reach.rust-lang.org). This awesome project connects people around the globe with Rust projects and - running in it's second year - has a lot of interesting learnings to present!

Reply:
cool, thanks!